draft-ietf-tls-esni.md

```diff
@@ -1048,13 +1048,8 @@ following string:
       8)
 ~~~
 
-As above, the payload of "encrypted_client_hello" is expected to be a
-`ClientECH` with `ClientECH.type` is `inner`. If this is not the case, the
-backend server MUST abort the handshake with an "illegal_parameter" alert.
-
-Note that, in case of HelloRetryRequest, the backend server confirms ECH
-acceptance twice: first In HelloRetryRequest, as described here; and then in the
-subsequent ServerHello, as described above.
+In the subsequent ServerHello message, the backend server sends the
```

Comment:
Why do we even need to mention this here since it has nothing to do with HRR? It's kind of confusing; it makes me wonder if there's a special accept_confirmation value that's used specifically when the server sent an HRR vs. when it didn't.

Reply:
> it makes me wonder if there's a special accept_confirmation value that's used specifically when the server sent an HRR vs. when it didn't.

Isn't this precisely the case now? There are two separate signals: one for ECH acceptance without HRR, and one for ECH acceptance with HRR.